controller/workitem.go

```diff
@@ -109,10 +110,39 @@ func (c *WorkitemController) List(ctx *app.ListWorkitemContext) error {
 		exp = criteria.And(exp, criteria.Equals(criteria.Field(workitem.SystemState), criteria.Literal(string(*ctx.FilterWorkitemstate))))
 		additionalQuery = append(additionalQuery, "filter[workitemstate]="+*ctx.FilterWorkitemstate)
 	}
+	if ctx.FilterCategory != nil {
+		var relationships []*category.WorkItemTypeCategoryRelationship
+		err := application.Transactional(c.db, func(tx application.Application) error {
+			// Load all workitemtypes related to the specific category
+			relationships, err = tx.Categories().LoadAllRelationshipsOfCategory(ctx, *ctx.FilterCategory)
+			log.Error(ctx, map[string]interface{}{
+				"category_id": *ctx.FilterCategory,
+				"err":         err,
+			}, "failed to list work items in category")
+			return err
+		})
+		if err != nil {
+			return jsonapi.JSONErrorResponse(ctx, errs.Wrap(err, fmt.Sprintf("failed to list work items in category %v", *ctx.FilterCategory)))
+		}
+		if len(relationships) == 0 {
+			log.Info(ctx, map[string]interface{}{
+				"category": *ctx.FilterCategory,
+			}, "no workitems belong to category: %s", *ctx.FilterCategory)
+			return nil
+		} else {
```

Comment:
You can simply place the body of the `else` branch below without nesting it in an `else` because you return from the `if` blocks above. Every level of nesting that can be avoided should be avoided for readability IMHO.